examples/backup_vm.py

```diff
@@ -390,11 +387,23 @@ def download_backup(connection, backup_uuid, args, incremental=False):
         if incremental and not has_incremental:
```

Comment:
`if incremental and not has_incremental:` - it's not you who wrote this code, IMHO very unreadable.

Reply:
This handle the case when the user asked to do an incremental backup, but the system cannot do an incremental backup for one of the disk. Would it be more clear if we use:

    if want_incremental and not can_incremental:

Reply:
The second suggestion will not work, since incremetnal is a property of the entire
backup, but has_incremental is a property of a disk download.

Reply:
A much more simple would be to just use a better name.
`has_incremental` --> `incremental_enabled` or even `incremental_enabled_for_disk`. This will make it a self-documented code :)
Also might be better change `available` to `enabled` in the `progress` next line.

Reply:
incremental_enabled or incremental_enabled_for_disk are incorrect,
this is not about feature enabled or not.

The purpose of this check is to find it we have incremental backup
for this disk, and download either incremental of full backup.